ui/src/pages/services/index.tsx

```diff
@@ -9,20 +9,189 @@ export default function ServicesPage() {
+              relevant services will fulfill all of your web development needs.
             </p>
             <Link to="/contact" className="cta">
               Contact Us
             </Link>
           </div>
         </div>
       </Section>
+      <Section>
+        <div className="grid">
+          <div>
+            <h1 className="margin-top-0">Custom Websites</h1>
+            <p>
+              Bold. Smooth. Modern. Showcase the best of your work, products, and services with a custom user interface
+              designed to flatter your brand. Our static sites can be as simple or complex as you need, with the option
+              to add in e-commerce with Shopify or manage content through your favorite CMS, including Wordpress,
+              Contentful, Sanity, and more. Interested in a stunning static website but need it right away? Check out
+              our semi-custom development services (link) to get going right out of the box. See our custom website work
+              in action for Tune & Flying Home Studio (link).
+            </p>
+            <ul>
+              <li>
+                Satisfy your niche needs in no time with rich support for APIs, widgets, and plugins. No matter what
+                you're looking for, from customer reviews to contact forms, our team uses JAMStack architecture to add
+                powerful functionality to your site without hurting its performance.
+              </li>
+              <li>
+                Tie in your preferred content management system (CMS) to seamlessly integrate new blog posts, photos, or
+                portfolio pieces into your site. We're all about convenience, and we strive to make content management
+                as simple as possible for all of our clients.
+              </li>
+              <li>
+                Reach an international audience. Our team’s experience with multilingual compatibility ensures that your
+                site will look and feel perfect for a diverse group of people, not just English speakers.
+              </li>
+            </ul>
+
+            <div>IMAGE</div>
+          </div>
+        </div>
+      </Section>
+      <Section>
+        <div className="grid">
+          <div>
+            <h1 className="margin-top-0">E-Commerce Integration</h1>
+            <p>
+              Rack up sale after sale with a custom Shopify theme. Designed for your unique needs, our e-commerce sites
+              offer all of the same benefits as our fast, adaptable brochure sites but with the added power of Shopify’s
+              seamless product listing and payment integration. Whether you’re looking to start from scratch or add to
+              your existing platform, our custom Shopify themes weave powerful e-commerce functionality right into your
+              stunning site. See our e-commerce work in action for ______.
```

Comment:
in this blank can we write Rachio and give an anchor tag that links to https://www.rachio.com